exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<p>dataset =&gt; data source</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>In that case it is dataset. Data source are usually something that provides continuously new data.</p><p> </p>

Reply:
Sorry, I should have been more specific. I referred to this "...  in order to demonstrate how the preprocessing tasks handle new data being added to a dataset". Here we are talking how the system works in general, when new data keeps coming. That's the data source.

Reply:
<p>ah ok</p>